### DISCUSSION TITLE
화면 공유가 안됩니다

### DISCUSSION THREAD

**Discussion by jihwanKo** [metamong-friends/metamong-friends.github.io#18] [제보하기]
화면공유를 시도하면 아래 캡쳐처럼 나와요.

![image](https://github.com/metamong-friends/metamong-friends.github.io/assets/49700905/fce7be13-751a-4834-b6f0-a2d6e2012658)

**Comment by jihwanKo** [comment]
[240503_Player.txt](https://github.com/metamong-friends/metamong-friends.github.io/files/15196717/240503_Player.txt)

로그입니다.

**Reply by shlee303** [reply]
확인해보겠습니다!

**Comment by ACT-TJ** [comment]
바쁘신데 로그까지 전달을 주셔서 감사드립니다. ㅠㅠ

로그상 캡처를 담당하고 있는 라이브러리 인식을 못하고 있는데요 ㅠㅠ
처음 발생한 현상이라 저희도 좀 더 재현과 로그분석을 해 보고 피드백 드리겠습니다.

잠시 시간되실 때 저희 인스톨러를 거치지 않은 빌드 실행파일로 프로님 pc 에서 실행을
해 보고자 하는데 괜찮으신 시간을 별도 연락드려 확인하고 도움 요청 드리도록 하겟습니다.

감사합니다!
